catalog/core/catalog-core-api/src/main/java/ddf/catalog/data/AttributeType.java

```diff
@@ -117,12 +115,18 @@
         XML,
 
         /**
+         * {@link Attribute}s of this {@link AttributeFormat} <b>must</b> have a value that is a
+         * Java {@link java.util.HashMap}.
+         */
+        HASHMAP,
```

Comment:
See my previous questions/comments on the need for this new type and further discussion as to the appropriate type it then might take.